enterprise/apps.py

```diff
@@ -17,7 +17,7 @@ class EnterpriseConfig(AppConfig):
 
     name = "enterprise"
     valid_extensions = [".png", ]
-    image_size = 250 * 1024
+    image_size = 4 * 1024
```

Comment:
We might want to make this a configurable value for other Open edX operators to manage.